packages/nextjs/pages/api/etl/index.ts

```diff
@@ -267,21 +271,29 @@ const getMovement = (current: number, comparison: number) => {
   return diff;
 };
 
-const getImpactIndex = (project: Metrics, weights: Metrics): number => {
+const getImpactIndex = (
+  project: Metrics,
+  weights: Metrics,
+  maxScoreByMetric: { [key in keyof Metrics]: number },
+): number => {
   let impact_index = 0;
-  const metricKeys = Object.keys(weights) as (keyof Metrics)[];
+  const metrics = Object.keys(maxScoreByMetric) as (keyof Metrics)[];
   let weightSum = 0;
-  for (const metric of metricKeys) {
+  for (const metric of metrics) {
     weightSum += weights[metric];
   }
   // If the weights don't add up to 100 then we expect they are each a 0-100 percentage and need to be adjusted
   const indPct = Math.round(weightSum) !== 100;
-  for (const metric of metricKeys) {
+  for (const metric of metrics) {
     const multiplier = indPct ? weights[metric] / weightSum : weights[metric];
-    // Normalize the weights to their percentage of the total
-    const projectMetric = project[metric as unknown as keyof Metrics];
-    if (projectMetric) {
-      impact_index += projectMetric * multiplier;
+
+    // Normalize score
+    const max = maxScoreByMetric[metric] || 0;
+    const normalizer = max !== 0 ? 100 / max : 0;
```

Comment:
This was taken from our [Impact Calculator implementation](https://github.com/BuidlGuidl/impact-calculator/blob/e6cd7f474abb0ad67df582c59748343ea0ef72eb/packages/nextjs/pages/api/impact/index.ts#L180)